R/zchunk_LA101.EIA_SEDS.R

```diff
@@ -1,20 +1,22 @@
 #' module_gcam.usa_LA101.EIA_SEDS
 #'
-#' Briefly describe what this chunk does.
+#' This chunk produces two ouput tables from the EIA state energy database:
+#'    L101.inEIA_EJ_state_S_F: Energy data by GCAM sector and fuel, state, and year; energy units in EJ, years from 1971-2010, includes only rows that have a defined sector and fuel
+#'    L101.EIA_use_all_Bbtu: Energy data by EIA sector and fuel code, GCAM sector and fuel, MSN, state, and year; energy units in Billion BTU, years from 1960-2011, includes all original data
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L101.EIA_use_all_Bbtu}, \code{L101.inEIA_EJ_state_S_F}. The corresponding file in the
 #' original data system was \code{LA101.EIA_SEDS.R} (gcam-usa level1).
-#' @details Describe in detail what this chunk does.
+#' @details See above
 #' @importFrom assertthat assert_that
 #' @importFrom dplyr filter mutate select
 #' @importFrom tidyr gather spread
-#' @author YourInitials CurrentMonthName 2017
-#' @export
-module_gcam.usa_LA101.EIA_SEDS_DISABLED <- function(command, ...) {
+#' @author AS April 2017
+#'
+module_gcam.usa_LA101.EIA_SEDS <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
     return(c(FILE = "gcam-usa/EIA_SEDS_fuels",
              FILE = "gcam-usa/EIA_SEDS_sectors",
```

Comment:
Right. Only put in `temp-data-inject/` if you're temporarily using old system data that will eventually be provided by another chunk.